pkg/kubectl/cmd/proxy.go

```diff
@@ -83,14 +87,63 @@ func NewCmdProxy(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringP("address", "", "127.0.0.1", "The IP address on which to serve on.")
 	cmd.Flags().Bool("disable-filter", false, "If true, disable request filtering in the proxy. This is dangerous, and can leave you vulnerable to XSRF attacks, when used with an accessible port.")
 	cmd.Flags().StringP("unix-socket", "u", "", "Unix socket on which to run the proxy.")
+	cmd.Flags().StringP("exec", "e", "", "Command to be execured with the proxy address.")
```

Comment:
"execured" (typo). Use "Command to be executed with the proxy address. {} will be replaced by the proxy URL." or something like that.